src/Rector/MethodCall/LumenRoutesStringActionToUsesArrayRector.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Laravel\Rector\MethodCall;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Scalar\String_;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Laravel\NodeAnalyzer\LumenRouteRegisteringMethodAnalyzer;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\Laravel\Tests\Rector\MethodCall\LumenRoutesStringActionToUsesArrayRector\LumenRoutesStringActionToUsesArrayRectorTest
+ */
+final class LumenRoutesStringActionToUsesArrayRector extends AbstractRector
+{
+    public function __construct(
+        private readonly LumenRouteRegisteringMethodAnalyzer $lumenRouteRegisteringMethodAnalyzer
+    ) {
+    }
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition(
+            'Changes action in rule definitions from string to array notation.',
+            [new CodeSample(<<<'CODE_SAMPLE'
+$router->get('/user', 'UserController@get');
+CODE_SAMPLE
+            , <<<'CODE_SAMPLE'
+$router->get('/user', ['uses => 'UserController@get']);
+CODE_SAMPLE)]
+        );
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class];
+    }
+
+    public function refactor(Node $node): ?Node
+    {
+        if (! $node instanceof MethodCall) {
+            return null;
+        }
+
+        if (! $this->lumenRouteRegisteringMethodAnalyzer->isLumenRoutingClass($node)) {
+            return null;
+        }
+
+        if (! $this->lumenRouteRegisteringMethodAnalyzer->isRoutesRegisterRoute($node->name)) {
+            return null;
+        }
+
+        $string = $node->args[1]->value;
```

Comment:
getArgs() here is fine, just don't call twice

Reply:
Done